Jenkinsfile

```diff
@@ -771,14 +778,8 @@ pipeline {
                         always {
                             recordIssues enabledForFailure: true,
                                          aggregatingResults: true,
-                                         id: "analysis-gcc-centos7",
-                                         tools: [ gcc4(pattern: 'centos7-gcc-build.log'),
-                                                  cppCheck(pattern: 'centos7-gcc-build.log') ],
-                                         filters: [ excludeFile('.*\\/_build\\.external\\/.*'),
-                                                    excludeFile('_build\\.external\\/.*') ]
-                        }
-                        success {
-                            sh "rm -rf _build.external"
+                                         tool: gcc4(pattern: 'centos7-gcc-build.log',
+                                                    id: "analysis-gcc-centos7")
```

Comment:
Is my assumption here that you can remove the filters because the deps and daos are built in separate steps so the build log only contains daos build output correct?

Reply:
Looking again, they are not built in separate steps, so the build log is going to contain all of the build output for the dependencies also, yes?  Do we really want to include that in the `recordIssues()` processing?

Reply:
The filters were stripping out _build.external which we haven't used for a long time, default goes into build/ however CI redirects this into /opt/daos so the filters were not masking anything anyway.

This is something which is (again accidently) improved by this PR, with this change the deps are always build or cached by the Dockerfile so this step is only ever building daos (in fact it uses --deps-build=no option) so there is no need for filters any more.

That's not withstanding that we had tools for cppcheck but have never used that tool and we compile with -Werror so any messages would cause a failure.  We could just as soon remove these recordIssues entirely.